webcentral/src/tools_over/data_import.py

```diff
@@ -52,7 +57,7 @@ class DataImportApp(DataImport):
         # "focus_en": "focus_en",
         "scale__en": "scale_en",
         # "lastUpdate_en": "lastUpdate_en",
-        "accessibility_en": "accessibility_en",
+        "accessibility__en": "accessibility_en",
```

Comment:
Yes this is correct. When importing the header fields from the structured data file to python I add "__en" to the fields coming from the worksheet "English". The keys in the dictionary specify, which of the imported columns should be imported as english translation. The corresponding dictionary value then specifies the ORM django attribute to import the data into. 
Do you think this logic is overengineered? I think in general it would be good to have such a mapping, because then the header names in the structured data files could be changed and the `data_import` could still be used.